.github/workflows/promote_branch.yaml

```diff
@@ -3,14 +3,14 @@ name: Promote branch
 on:
   workflow_dispatch:
     inputs:
-      target-branch:
-        description: The name of the branch to be promoted
+      promotion-type:
+        description: Which branch is being promoted where
```

Comment:
Maybe "the type of promotion to perform" ?

Reply:
fixed, thanks